[2021-10-16 13:05:57,562] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_2 scheduled__2021-10-16T12:48:00+00:00 [queued]>
[2021-10-16 13:05:57,622] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_2 scheduled__2021-10-16T12:48:00+00:00 [queued]>
[2021-10-16 13:05:57,622] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:05:57,622] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-16 13:05:57,622] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:05:57,668] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): get_gross_year_2> on 2021-10-16 12:48:00+00:00
[2021-10-16 13:05:57,685] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'get_gross_year_2', 'scheduled__2021-10-16T12:48:00+00:00', '--job-id', '531', '--raw', '--subdir', 'DAGS_FOLDER/fourth_dag.py', '--cfg-path', '/tmp/tmp5shj2_jt', '--error-file', '/tmp/tmpxlxb50xn']
[2021-10-16 13:05:57,680] {standard_task_runner.py:52} INFO - Started process 1037 to run task
[2021-10-16 13:05:57,687] {standard_task_runner.py:77} INFO - Job 531: Subtask get_gross_year_2
[2021-10-16 13:05:57,790] {logging_mixin.py:109} INFO - Running <TaskInstance: fourth_dag.get_gross_year_2 scheduled__2021-10-16T12:48:00+00:00 [running]> on host e03fb9e01fd8
[2021-10-16 13:05:57,921] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fourth_dag
AIRFLOW_CTX_TASK_ID=get_gross_year_2
AIRFLOW_CTX_EXECUTION_DATE=2021-10-16T12:48:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-16T12:48:00+00:00
[2021-10-16 13:05:57,956] {python.py:152} INFO - Done. Returned value was: ['$20.10 ', '2010\n']
[2021-10-16 13:05:58,089] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=get_gross_year_2, execution_date=20211016T124800, start_date=20211016T130557, end_date=20211016T130558
[2021-10-16 13:05:58,172] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-16 13:05:58,324] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
